examples/example7a_SAS/example7a.py

```diff
@@ -0,0 +1,49 @@
+# SPDX-FileCopyrightText: 2022 UChicago Argonne, LLC
+# SPDX-License-Identifier: MIT
+
+#########################################################
+# Example problem of runing SAS4A/SASSY-1 with WATTS.
+# This is a steady-state, single pin problem with uniform 
+# dimensions and constant properties. The fuel pin has a
+# constant linear heat generation and the boundaries of 
+# the problem are the top and bottom of the fuel pin. 
+# Sodium coolant enters from the bottom. The fuel pin is
+# separated into a bottom unheated section, fuel section,
+# gas plenum, and top unheated section. This problem DOES
+# NOT use the PRIMAR4 module.
+#########################################################
+
+from math import cos, pi
+import os
+import watts
+from astropy.units import Quantity
+
+
+params = watts.Parameters()
+
+# Channel params
+params['sas_version'] = 5.5
+params['tmax'] = 400.0 # maximum problem time in s
+params['flow_per_pin'] = 0.15 # kg/s
+params['total_reactor_power'] = Quantity(182, "MW") # 182000000.0 W
+params['betai_1'] = 2.0E-04 # Effective delayed neutron fraction
+params['betai_2'] = 1.0E-03
+params['betai_3'] = 1.2E-03
+params['betai_4'] = 2.5E-03
+params['betai_5'] = 1.5E-03
+params['betai_6'] = 5.0E-04
+
+params.show_summary(show_metadata=False, sort_by='key')
+
+# SAS Workflow
+# Set SAS directory as sas_dir
+sas_plugin = watts.PluginSAS('sas_template') # Show all the output
+sas_dir = "/path/to/SAS" # Path to SAS directory
+sas_plugin.sas_exec = sas_dir + "/bin/sas-5.5-Darwin-x86_64.x" # SAS executable
+sas_plugin.conv_channel = sas_dir + "/Tools/macOS/CHANNELtoCSV-Darwin-x86_64.x" # SAS utility to convert "CHANNEL.dat" files to ".csv"
+sas_plugin.conv_primar4 = sas_dir + "/Tools/macOS/PRIMAR4toCSV-Darwin-x86_64.x" # SAS utility to convert "PRIMAR4.dat" files to ".csv"
+sas_result = sas_plugin.workflow(params)
```

Comment:
If we do this, we may "force" the user to provide SAS_DIR that will contain `sas.x`, `CHANNELtoCSV.x` and `PRIMAR4toCSV.x`

Reply:
Thanks for the modification. I think these 3 lines can now be removed.
```
 sas_plugin.sas_exec = sas_dir + "/sas.x" # SAS executable
 sas_plugin.conv_channel = sas_dir + "/CHANNELtoCSV.x" # SAS utility to convert "CHANNEL.dat" files to ".csv"
 sas_plugin.conv_primar4 = sas_dir + "/PRIMAR4toCSV.x" # SAS utility to convert "PRIMAR4.dat" files to ".csv"
```

Reply:
@nstauff I don't think I am following you here. If we remove the paths to `sas_exec` and the two utilities then the code wouldn't run, right? My understanding is that in the environment, "SAM_DIR" is only set to the directory containing these three files so the specific paths to the files still need to specified explicitly.

Reply:
This is right with the current implementation. Please check the `plugin_pyarc` and `example1c` for another example (different from MOOSE). On the MOOSE plugin, we need to provide the MOOSE_DIR and the executable because we are specifying which MOOSE app needs to be run. On the PyARC, we can directly provide link to PyARC directory with PYARC_DIR, and then we don't need to specify anymore the executable (but we still have the option). 
Maybe I am overthinking it and what you have will be good. I will be interested to hear the review from @paulromano!

Reply:
Thanks for the clarification. I will update the PR to follow PyARC's implementation. I think it's better this way than the current implementation.